src/measure_theory/vector_measure.lean

```diff
@@ -815,4 +815,65 @@ end
 
 end vector_measure
 
+namespace signed_measure
+
+open vector_measure
+
+open_locale measure_theory
+
+/-- The underlying function for `signed_measure.to_measure`. -/
+def to_measure' (s : signed_measure α)
+  (i : set α) (hi₁ : measurable_set i) (hi₂ : 0 ≤[i] s)
+  (j : set α) (hj₁ : measurable_set j) : ℝ≥0∞ :=
+@coe ℝ≥0 ℝ≥0∞ _ ⟨s.restrict i j,
+begin
+  rw [s.restrict_apply hi₁ hj₁, ← s.restrict_apply hi₁ hj₁],
+  exact le_trans (by simp) (hi₂ j hj₁),
+end⟩
+
+/-- Given a signed measure `s` and a positive measurable set `i`, `to_measure`
+provides the measure, mapping measurable sets `j` to `s (i ∩ j)`. -/
+def to_measure (s : signed_measure α) (i : set α) (hi₁ : measurable_set i) (hi₂ : 0 ≤[i] s) :
```

Comment:
Can you add a statement that links this with `to_signed_measure`?

Reply:
I've added the lemmas `to_signed_measure_to_measure` and `to_measure_to_signed_measure`